terraform/validate.sh

```diff
@@ -1,8 +1,8 @@
 rm -rf .terraform
-terraform init -input=false -backend=false -no-color "$module_path"
+terraform "$chdir" init -input=false -backend=false -no-color "$config_path"
```

Comment:
it seems as though this is coming out as:

```bash
terraform -chdir=cluster init -input=false -backend=false -no-color ''
```

and the single quotes at the end are breaking it.